src/components/media/_media.scss

```diff
@@ -60,11 +60,6 @@ $mediaItemSecondaryColor: $grayPrimary;
     min-width: 0;
     position: relative;
     flex: 1 1 0;
-
-    &--centered {
-      display: flex;
-      align-items: center;
-    }
```

Comment:
not used
https://github.com/search?utf8=%E2%9C%93&q=org%3Abrainly+sg-media__wrapper--centered&type=Code
and also I tried search it manually